FerramAerospaceResearch/FARAPI.cs

```diff
@@ -401,6 +401,24 @@ public static Vector3 VesselAerodynamicTorque(Vessel v)
         {
             return VesselFlightInfo(v)?.InfoParameters.aerodynamicTorque ?? Vector3.zero;
         }
+
+        /// <summary>
+        /// Returns the current aerodynamic force being experienced by the active vehicle in world space
+        /// </summary>
+        /// <returns>The force on the vessel in world space</returns>
+        public static Vector3 ActiveVesselAerodynamicForce(Vessel v)
```

Comment:
https://github.com/dkavolis/Ferram-Aerospace-Research/blob/ec14134c56a450569cf5c4fcf2b00734d2b7aaaa/FerramAerospaceResearch/FARAPI.cs#L409
https://github.com/dkavolis/Ferram-Aerospace-Research/blob/ec14134c56a450569cf5c4fcf2b00734d2b7aaaa/FerramAerospaceResearch/FARAPI.cs#L418

No need for `Vessel` arguments